exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/QueryWrapper.java

```diff
@@ -48,13 +51,21 @@
   private final String queryType;
   private final int autoLimitRowCount;
 
+  @Nullable
+  private final Map<String, Object> options;
+
   private static MemoryMXBean memMXBean = ManagementFactory.getMemoryMXBean();
 
   @JsonCreator
-  public QueryWrapper(@JsonProperty("query") String query, @JsonProperty("queryType") String queryType, @JsonProperty("autoLimit") String autoLimit) {
+  public QueryWrapper(
+    @JsonProperty("query") String query,
+    @Nullable @JsonProperty("queryType") String queryType,
```

Comment:
Please remove nullable annotation, `javax.annotation` is a separate dependency and it is built in.